client/src/components/Commits.js

```diff
@@ -87,13 +118,15 @@ class Commits extends React.Component {
 }
 
 // prettier-ignore
-const mapStateToProps = ({ isLoading, commits, error, branch,page,buildData}) => ({
+const mapStateToProps = ({ isLoading, commits, error, branch,page,buildData,bsid,build}) => ({
```

Comment:
Missing space after commas and extra space after bracket

Reply:
prettier-ignore is the culprit :)